### ISSUE TITLE
`dropdown-options` always open and lacks a11y

### ISSUE THREAD

**Issue by Link2Twenty** [forem/forem#13318]
**Describe the bug**
Drop down options in for items in the "OTHER THINGS YOU CAN DO" section of the mod panel are always open.
While investigating the cause I noticed there were no aria attributes

ref: <https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html>

**To Reproduce**
- Go to a post
- Open the mod panel
- See the accordion appears open
- Click the accordion and see nothing changes

**Expected behaviour**
The accordion should open and close on click as well as modify it's aria attributes

**Screenshots**
![dev__dropdown-options](https://user-images.githubusercontent.com/3534427/114010029-9be73980-985b-11eb-98d8-197710b3ba5c.gif)

**Desktop (please complete the following information):**

- OS, version: Windows 10
- Browser, version: 89.0.4389.114

**Smartphone (please complete the following information):**
N/A

**Additional context**
The `display: flex` here 
https://github.com/forem/forem/blob/009c14653ab84da91cf278400350503e056bef29/app/assets/stylesheets/views/mod-actions.scss#L367-L372

Overrides the `display: none` generated here
https://github.com/forem/forem/blob/009c14653ab84da91cf278400350503e056bef29/app/assets/stylesheets/config/_generator.scss#L600

**Solutions:** __for visible bug__
- add hidden to `mod-actions` CSS for higher specificity
- create crayons-accordion that solves all these issues
- use summary/details instead of doing JS magic
